lib/beaker/hypervisor/hcloud.rb

```diff
@@ -69,6 +77,25 @@ def create_ssh_key
       hcloud_ssh_key
     end
 
+    # we need to save the date as unix timestamp. Hetzner Cloud labels only support:
+    # "alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between."
+    # https://docs.hetzner.cloud/#labels
+    def vm_deletion_date
+      (DateTime.current + @delete_vm_after.hours).to_i.to_s
```

Comment:
You can also use native Ruby to avoid the ActiveSupport dependency
```suggestion
      (DateTime.now + (@delete_vm_after /24)).to_i.to_s
```

Reply:
ah right, that works as well. We pull in activesupport anyways because it's a dependency for hcloud. So should we use it here?